VultisigApp/VultisigApp/Services/BalanceService.swift

```diff
@@ -20,78 +20,92 @@ class BalanceService {
     private let kuji = KujiraService.shared
     private let maya = MayachainService.shared
     private let dot = PolkadotService.shared
-
+    
+    private var cache: ThreadSafeDictionary<String, (data: Bool, timestamp: Date)> = ThreadSafeDictionary()
```

Comment:
can you remove the cache? as discussed , we could just keep when is the last time user check balance , if it is less than 1 minutes , we could skip it

Reply:
yes, I've done that i just forgot to push